cluster/manifests/flannel/daemonset.yaml

```diff
@@ -100,3 +119,7 @@ spec:
       - name: run
         hostPath:
           path: /run
+      - hostPath:
+          path: /lib/tc
+          type: ""
+        name: lib-tc
```

Comment:
What is this volume used for?

Reply:
For making "tc" possible to run

Reply:
Maybe we could put this folder in the Dockerfile if it's just needed by `tc`. Then the configuration would be a bit less confusing :) But not a big deal.

Reply:
But if anyone else adds also something to tc, then it does not have the xtables.lock, which might do more harm. I think it's good to have it in the hostpath.